include/exception.h

```diff
@@ -41,7 +41,7 @@ class PHPCPP_EXPORT Exception : public std::exception
      *  Constructor
      *  @param  &string
      */
-    Exception(std::string message, int code = 0) : std::exception(), _message(std::move(message)), _code(code) {}
+    Exception(const std::string& message, int code = 0) : std::exception(), _message(std::move(message)), _code(code) {}
```

Comment:
Well, running the tests now, I see that the actual bug was somewhere else — because now the master branch passes that test.

As far as I can tell, the issue had to do something with the lifetime of the temporary string variable. `message` was passed by value, this caused it to be copied when the exception was allocated, and according to the backtrace, `Php::Exception` constructor got the broken copy of the message. Experiments showed that either of the methods had worked: a) pass the temporary by const reference (this delays the destruction of the temporary), b) get rid of `std::move`. (B) would add more overhead (two copies by value), that is why I used (A) way.

Basically now the code is almost the same as it used to be, only the temporary variable gets destroyed a bit later, and copy constructor for `message` called later as well.

Because the test now passes, this PR can probably be closed.